rq/utils.py

```diff
@@ -297,3 +297,25 @@ def split_list(a_list, segment_size):
     """
     for i in range(0, len(a_list), segment_size):
         yield a_list[i:i + segment_size]
+
+
+def get_call_string(func_name, args, kwargs, truncate=False, maxlen=75):
```

Comment:
`maxlen` -> `max_length`.

If we already have `max_length`, we can get rid of `truncate` argument. We can just default `max_length` to `None`.

Reply:
:+1: